drivers/remote/driver.go

```diff
@@ -247,6 +247,30 @@ func (d *driver) Type() string {
 	return d.networkType
 }
 
+// DiscoverNew is a notification for a new discovery event, such as a new node joining a cluster
+func (d *driver) DiscoverNew(dType driverapi.DiscoveryType, jsonStr string) error {
+	if dType != driverapi.NodeDiscovery {
+		return fmt.Errorf("Unknown discovery type : %v", dType)
+	}
+	notif := &api.DiscoveryNotification{
+		DiscoveryType:     dType,
+		DiscoveryJSONData: jsonStr,
```

Comment:
LOL. Infact I originally had DiscoveryData as an interface{}. But finally ended up with a JsonString assuming that the drivers would actually prefer it due to the loosely coupled nature of the data and easier to read/document.

But if you prefer it to be an `interface{}`, thats fine by me.